lib/cli/src/helpers.js

```diff
@@ -316,6 +317,12 @@ export function addToDevDependenciesIfNotPresent(packageJson, name, packageVersi
 export function copyTemplate(templateRoot, storyFormat) {
   const templateDir = path.resolve(templateRoot, `template-${storyFormat}/`);
   if (!fs.existsSync(templateDir)) {
+    // Fallback to CSF plain first, in case format is typescript but template is not available.
+    if (storyFormat === supportedStoryFormats.CSF_TYPESCRIPT) {
```

Comment:
WDYT about a warning here? Something like `No TS boilerplate is available for ${framework}, falling back to JS instead.`

Reply:
I thought of doing that, but there are a few problems:

1 - This method is not aware of the framework, unless I do a not so pretty code like `templateRoot.split('/').pop()`, which would be using the template folder, resulting in a message like:
`No TS boilerplate is available for SFC_VUE, falling back to JS instead.`.
It's not great because the folders could have names that are not interesting for the users, like SFC_VUE, or WEBPACK_REACT.
To pass a proper name the method would need to be changed, so that it would receive as parameter from every template generator function.

2 - There is inconsistency in some of the template folders. For instance Angular has `template-csf`, even though it's typescript. As a result, for angular this would be the output:
```
• Detecting project type. ✓      
No TS boilerplate is available, falling back to JS instead. 
• Adding storybook support to your "Angular" app. ✓
```
Leaving the user a bit confused.

I'm up for fixing this, by renaming all folders properly if you want 👍

Reply:
Nah, don’t worry about it. Just a suggestion!